components/builder-web/app/shared/project-settings/project-settings.component.html

```diff
@@ -1,7 +1,8 @@
 <div class="project-settings-component">
   <div class="connect" *ngIf="!project && !connecting">
-    <button mat-raised-button color="primary" class="button" (click)="connect()" [disabled]="!hasPrivateKey">
-      Connect a plan file
```

Comment:
Does this completely remove the capability to connect a plan file after this PR is merged?
If so, we either need a stopgap or a follow up PR ready to go that enables plan connection..

Reply:
Good Catch: I believe it's going in to the updated Package Settings Tab, which i'm working on now.  I'll throw a DO-NOT-MERGE on this for now.  I think I may need to align with @scottopherson, as we _might_ need to merge in a specific order so that we don't step on each others toes too much.

Reply:
:+1: